### DISCUSSION TITLE
Theia IDE 1.51 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13870] [Release Announcements] [answered]
The new version 1.51.0 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.51.0/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.51.0/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.51.0/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference `updates.channel` to `preview`. Please respond here when you were able to test the preview without finding blockers, by commenting with a ✔️. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by tsmaeder** [comment]
I've been using the release since this morning and I've noticed nothing unusual.

**Comment by sgraband** [comment]
Used it the whole day and experienced no issues :+1:

**Comment by JonasHelming** [comment]
Works well for me!

**Comment by tsmaeder** [comment]
As I haven't heard or experienced any problems, I'm all for promoting the preview this afternoon CEST. Objections?

**Reply by jfaltermeier** [reply]
I'll go ahead and promote the IDE

**Reply by jfaltermeier** [reply, marked as answer]
1.51.0 is promoted to the stable channel

**Comment by JonasHelming** [comment]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/07/04/eclipse-theia-1-51-release-news-and-noteworthy/)
